scripts/verify-16kb-alignment.sh

```
@@ -0,0 +1,133 @@
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    readelf="$ndk_path/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-readelf"
+  fi
+
+  if [ ! -f "$readelf" ]; then
+    echo "❌ Error: llvm-readelf not found at $readelf"
+    echo "💡 Make sure ANDROID_NDK_ROOT is set or NDK is installed at $ANDROID_HOME/ndk/29.0.14206865"
+    return 1
+  fi
+
+  # Check LOAD segments alignment
+  local alignments=$($readelf -l "$lib_path" | grep "LOAD" | awk '{print $NF}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)